### ISSUE TITLE
Check Constraint

### ISSUE THREAD

**Issue by andrew000** [tortoise/tortoise-orm#614]
### How about creating a Check Constraint when creating a table through the Model?

Link to PGSQL documentation: https://postgrespro.com/docs/postgresql/12/ddl-constraints#DDL-CONSTRAINTS-CHECK-CONSTRAINTS

![image](https://user-images.githubusercontent.com/11490628/104827420-fd53c980-5865-11eb-9e75-2162c61fde3f.png)

``` python3
class Products(Model):
    product_no = IntField(pk=True)
    name = CharField(max_length=255)
    price = IntField()

    class Meta:
        table = "products"
        table_args = (CheckConstraint('price > 0'),  # CheckConstraint as in SQLAlchemy
                      )
```

**Comment by WisdomPill** [comment]
Hello @andrew000.

Can it be a validator like django has MaxValueValidator and MinValueValidator?

Then pass a list of Validators to a field, that could be custom in some cases, but min and max should be the bare minimum to start with

**Comment by WisdomPill** [comment]
I think this can be done with validators that have been recently released with version `0.16.20`, is that right @long2ice ?

**Comment by long2ice** [comment]
Yes
